### PR TITLE
Fix BSA 183 - Add missing translations plugins 

### PR DESCRIPTION
**Pull request by habemuscode**
**Related to**

https://oat-sa.atlassian.net/browse/BSA-183

**Related to this PR**

https://github.com/oat-sa/ckeditor-dev/pull/18

**Description**

This PR is to allow dynamic translations of CKEditor based on selected language by the user

**AC**

- Translate only `CKeditor` keys of specific plugins in NL-DE-FR languages

**Note**

For other languages I create this tech debt issue https://oat-sa.atlassian.net/browse/TAO-10422

### CODE REVIEW

**Comment by habemuscode** [comment]
Copy/paste after build of `CKeditor` files from this PR: https://github.com/oat-sa/ckeditor-dev/pull/18

File: .idea/misc.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (bziondik) [inline .idea/misc.xml:1]: please, remove `.idea` files and add `.idea` to `.gitignore`

Comment (habemuscode) [reply .idea/misc.xml:1]: Done, I will not add `node_modules` for now to `.gitignore`, because not needed here, will never appear that folder.

**Comment by habemuscode** [comment]
@bziondik Thanks! I will do in this PR: https://github.com/ckeditor/ckeditor4/pull/4172 Add all of you when finish

**Comment by bziondik** [comment]
> @bziondik Thanks! I will do in this PR: [ckeditor/ckeditor4#4172](https://github.com/ckeditor/ckeditor4/pull/4172) Add all of you when finish

You add a lot of changes from our fork to the original  `ckeditor4` repo. You need to create just small PR with needed translations without our own changes

**Comment by habemuscode** [comment]
@bziondik Update link to -> https://github.com/oat-sa/ckeditor-dev/pull/19. Forget to change base branch on create PR :(

**Comment by habemuscode** [comment]
Merged, validated in my local BOSA environment before merge

**Pull request merged**